docs/content/2.guide/8.lazy-load-translations.md

```diff
@@ -67,3 +68,86 @@ export default defineNuxtConfig({
 Currently, `json`,`json5` and `yaml` are supported only.
 
 ::
+
+## Multiple files lazy loading
+
+The `files` can load lazily multiple files.
+
+This is useful because it is efficient to manage multiple files that define only differences without duplicating locale messages.
+
+For example, let’s think for the case of Spanish language supporting. There are **79 Spanish-speaking countries** according to [wikipedia](https://en.wikipedia.org/wiki/Spanish_language)!
```

Comment:
19 countries listed on ISO (official)

Reply:
Still 79, it should be 19, read the article again, these are estimated people who speak Spanish in those countries: in Spain there are people who speak Japanese but in Spain Japanese is not official, you don't have to count Spain as speaking countries japanese ;)

Check `Geographical distribution`, it just mention 20 countries, you're using `Spanish speakers by country` entry.

Reply:
upps (updated to 15), why showing changes shows previouws version?